documentation/configuration.markdown

```diff
@@ -93,6 +93,18 @@ http-client: {
     #                TLS_RSA_WITH_AES_256_CBC_SHA,
     #                TLS_RSA_WITH_AES_128_CBC_SHA256,
     #                TLS_RSA_WITH_AES_128_CBC_SHA]
+    
+    # The amount of time, in milliseconds, that an outbound HTTP connection
+    # will wait for data to be available before closing the socket. If not
+    # defined, defaults to 20 minutes. If 0, the timeout is infinite and if
+    # negative, the value is undefined (system default).
+    #idle-timeout-milliseconds: 1200000
```

Comment:
Same comment here as for the other `idle-timeout-milliseconds` reference in "puppet-server.conf.sample".